.github/ISSUE_TEMPLATE/2-feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
can we auto label with `type: feature request`? that would be awesome

Reply:
Yeah that would be great

Reply:
@bergquist Not sure how to work this into our issue triage workflow in a good way. Perhaps we want a separate label that we can use here that we can change into the real one if we deem it was correctly tagged.

Reply:
Or we could add a tag for incoming issues, we basically just need a way to find untriaged issues.

Reply:
Sounds like a good idea